author: Andres Salomon <dilinger@debian.org>

The #define LIBCP_STDATOMIC_H is meant to block inclusion of <stdatomic.h>,
which conflicts w/ chromium's <atomic>. However, it doesn't actually block
it (maybe newer clang uses a different inclusion guard symbol?). This is
what clang-19 needs to block stdatomic.h.

--- a/third_party/pthreadpool/chromium/jobs.cc
+++ b/third_party/pthreadpool/chromium/jobs.cc
@@ -18,6 +18,7 @@
 // crbug.com/391750836.
 #include <atomic>
 #define _LIBCPP_STDATOMIC_H
+#define __CLANG_STDATOMIC_H
 using namespace std;
 
 // Configuration header.
